ogr/abstract.py

```diff
@@ -189,14 +189,21 @@ def __str__(self) -> str:
 
     @staticmethod
     def create(
-        project: Any, title: str, body: str, private: Optional[bool] = None
+        project: Any,
+        title: str,
+        body: str,
+        private: Optional[bool] = None,
+        labels: Optional[list] = None,
+        assignees: Optional[list] = None,
     ) -> "Issue":
```

Comment:
Would be nice to add the type of the elements -- `Optional[List[str]]` probably.

Note that `list[str]` syntax is only for python3.9+

Reply:
Hey, I tried it, the current `tox` config uses py36 and py37, Tests were failing there any way to silent that or has a way around to support both?

Reply:
Might be some issues with my config🤷🏻‍♂️, I have fixed it!